extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -541,7 +541,7 @@ export class AmpAnalytics extends AMP.BaseElement {
     if (!key) {
       return {name: '', argList: ''};
     }
-    const match = key.match(/^(?:([^ ]*)(\([^)]*\))|.+)$/);
+    const match = key.match(/(?:^([^ ]*)(\([^)]*\)$)|[^]+)/);
```

Comment:
hmm. That works. For some reason, I missed that during testing. Will update.


for posterity: 
From https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp#Special_characters_meaning_in_regular_expressions : 

> Note that the m multiline flag doesn't change the dot behavior. So to match a pattern across multiple lines, the character set [^] can be used (if you don't mean an old version of IE, of course), it will match any character including newlines.